src/libraries/System.Security.Cryptography.Algorithms/README.md

```diff
@@ -0,0 +1,11 @@
+# System.Security.Cryptography.Algorithms
+
+This assembly no longer contains any code.  It is provided only to permit type unification for libraries built against previous versions of .NET.
```

Comment:
nit: redundant space?

```suggestion
This assembly no longer contains any code. It is provided only to permit type unification for libraries built against previous versions of .NET.
```

Reply:
It [just shows my age](https://www.thesaurus.com/e/writing/is-there-1-space-or-2-after-a-period/).  I'll give up on my double-spaced sentence separators when a linter pries it from my cold, dead fingers. :smirk: